tests/extensions/aria_extension_tosca/simple_nfv_v1_0/test_profile.py

```diff
@@ -0,0 +1,20 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
So basically we have no unittests for the parser

Reply:
You are looking here specifically at the test to make sure the NFV profile is valid. This is not where the unit tests for the parser lie.

Reply:
Maybe i don't yet understand the entire testing suite, but all of the tests test the full path. I think i need an introduction into the new parser testing mechanism